### PR TITLE
WIP: new upstream release

### PR DESCRIPTION
**Pull request by jbweston**
DO NOT MERGE THIS YET: This is just to check that the builds function. Upstream source set to my personal website to avoid bureaucracy, will revert to official kwant website when ready to merge.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jbweston** [comment]
@basnijholt why are travis, circleCI etc. still running? I thought that the new azure pipelines would mean that the old stuff doesn't need to be done anymore

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  url: https://downloads.kwant-project.org/kwant/kwant-{{ version }}.tar.gz
-  sha256: e43e9ed928ff3e7cbe8e7b9e4b671871a36c72dd1000548746098af89f102a67
+  url: http://josephweston.org/uploads/kwant-{{ version }}.tar.gz
+  sha256: eb487b7d1f4f859255c5ec1901b029a792e1e03b918f1a169c57f612a48bc366
   patches:
     - skip_tests.patch  # [win]
 
 build:
   skip: true  # [py2k]
   skip: true  # [py==37]
```
Comment (akhmerov) [inline recipe/meta.yaml]: We shouldn't be skipping `py==37` anymore.

**Comment by basnijholt** [comment]
The process of completely moving to Azure is not yet complete AFAIU.

**Comment by akhmerov** [comment]
Also pinning `numpy <= 1.11` seems obsolete

**Comment by basnijholt** [comment]
You should always build with the oldest numpy that is supported

File: recipe/meta.yaml (hunk 1 above)
Comment (jbweston) [reply recipe/meta.yaml]: this was added as part of some PR, I `git blame`d it but the commit message was just "disable Python 3.7" I left it as is for now, but was planning to re-enable once the rest of the builds pass

**Comment by jbweston** [comment]
@conda-forge-admin please re-render

File: recipe/meta.yaml (hunk 1 above)
Comment (basnijholt) [reply recipe/meta.yaml]: It was because Sympy 1.1 wasn't build for Python 3.7.

**Comment by jbweston** [comment]
Argh, I opened this from `master`, not a feature branch, so problems. Will close/open a new PR

**Pull request closed without merging**